README.md

```diff
@@ -19,38 +19,53 @@ This repository uses [Microsoft Rush](https://rushjs.io/) to manage multiple pac
 - npm 10.x or later
 - Git
 
-### Setup
+### 🚀 First-Time Setup
 
-**Quick Setup (Recommended):**
+This monorepo uses git worktrees for external dependencies. Before running Rush commands, you need to set up the worktrees:
 
+**Linux/macOS:**
 ```bash
 # Clone the repository
 git clone https://github.com/bosconian-dynamics/openfront-projects.git
 cd openfront-projects
 
 # Run the setup script
-./setup.sh        # For Linux/macOS
-# or
-./setup.ps1       # For Windows/PowerShell
+./scripts/setup-worktrees.sh
+rush update
+rush build
 ```
 
-**Manual Setup:**
-
-```bash
+**Windows (PowerShell):**
+```powershell
 # Clone the repository
 git clone https://github.com/bosconian-dynamics/openfront-projects.git
 cd openfront-projects
 
-# Install Rush globally
-npm install -g @microsoft/rush
-
-# Install all package dependencies
+# Run the setup script
+.\scripts\setup-worktrees.ps1
 rush update
-
-# Build all packages
 rush build
 ```
 
+See [docs/WORKTREE_WORKFLOW.md](docs/WORKTREE_WORKFLOW.md) for detailed information about working with worktrees.
+
+### Alternative: Quick Setup Script
+
+**Quick Setup (Uses existing setup scripts):**
+
+```bash
+# Clone the repository
+git clone https://github.com/bosconian-dynamics/openfront-projects.git
+cd openfront-projects
+
+# Run the setup script
+./setup.sh        # For Linux/macOS
+# or
+./setup.ps1       # For Windows/PowerShell
+```
+
+Note: The `setup.sh` and `setup.ps1` scripts will need to be updated to call the worktree setup script.
```

Comment:
I don't understand what this is about. You yourself have modified the setup scripts to call the worktree setup script in this very PR...